packages/headless/src/controllers/commerce/triggers/headless-notify-trigger.ssr.ts

```diff
@@ -1,11 +1,11 @@
-import {SharedControllerDefinitionWithoutProps} from '../../../app/commerce-ssr-engine/types/common';
+import {SearchAndListingControllerDefinitionWithoutProps} from '../../../app/commerce-ssr-engine/types/common';
 import {NotifyTrigger} from '../../core/triggers/headless-core-notify-trigger';
 import {buildNotifyTrigger} from './headless-notify-trigger';
 
 export type {NotifyTriggerState} from '../../core/triggers/headless-core-notify-trigger';
 export type {NotifyTrigger};
 export interface NotifyTriggerDefinition
-  extends SharedControllerDefinitionWithoutProps<NotifyTrigger> {}
+  extends SearchAndListingControllerDefinitionWithoutProps<NotifyTrigger> {}
```

Comment:
I think this one should be search only: notify triggers on listing pages don't really make sense as there isn't a query.